api/src/main/java/org/openmrs/validator/UserValidator.java

```diff
@@ -125,15 +125,13 @@ public void validate(Object obj, Errors errors) {
 				}
 			}
 			
-			if (!StringUtils.isBlank(user.getEmail())) {
-				if(!isEmailValid(user.getEmail())) {
-					errors.rejectValue("email", "error.email.invalid");
-				}
-			}
-			
-			ValidateUtil.validateFieldLengths(errors, obj.getClass(), "username", "systemId", "retireReason");
-		}
-	}
+			 if (!StringUtils.isBlank(user.getEmail()) && !isEmailValid(user.getEmail())) {
+	        errors.rejectValue("email", "error.email.invalid");
+	      } else {
+	        ValidateUtil.validateFieldLengths(errors, obj.getClass(), "username", "systemId", "retireReason");
+	      }
+	   	}
+   }
```

Comment:
And here.
![image](https://user-images.githubusercontent.com/9057882/91315694-496b7800-e7b8-11ea-9c9e-7ec08eda0bb6.png)